src/views/MarketFilter.tsx

```diff
@@ -36,27 +39,53 @@ export const MarketFilter = ({
   hideNewMarketButton?: boolean;
   searchPlaceholderKey?: string;
   compactLayout?: boolean;
+  unlaunchedMarketToggle?: {
```

Comment:
as a pattern this isn't great because most callers will pass a fresh object which dooms component level memoization and makes it so you get re-rendered whenever the parent does. In our case it doesn't really matter since this function isn't memoed and we don't really care about memoization anyway.

Reply:
I dunno I still kind of prefer we don't but it is purely a style thing at this point.

Reply:
makes sense!